app/app/models/cbv_flow_invitation.rb

```diff
@@ -1,6 +1,5 @@
 class CbvFlowInvitation < ApplicationRecord
-  EMAIL_REGEX = URI::MailTo::EMAIL_REGEXP
-
+  EMAIL_REGEX = /\A([\w+\-].?)+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i
```

Comment:
Sigh, this is a classically hard problem to solve. This regex would be invalid for domains that have subdomains and numbers, i.e. "foo@test1.test2.com". That seems unlikely so I'll approve this but my point here is that it's going to be hard to figure out all the edge cases/possibilities.

Personally the best easy validation I've seen is just to verify that the email address has an "@" in it and a "." following that, i.e. something like.:

```ruby
/\A(.*)@(.*)\.(.*)\z/
```

Reply:
![Screenshot 2024-09-25 at 2 05 23 PM](https://github.com/user-attachments/assets/a321377b-ffda-4636-9b45-48d110634860)

That regex doesn't seem to satisfy some email addresses that are clearly invalid.

Reply:
What I meant to say is that the point of this regex is just to prevent common caseworker typos, not to perfectly validate an email address format. The worst case scenario is we have a false positive where the regex says an email address is invalid, but it actually is valid, and then the caseworker gets stuck. So at the minimum, I think it would be good to handle the edge case I noticed by changing the regex to allow numbers and dashes in the domain components, since those may happen.

```suggestion
  EMAIL_REGEX = /\A([\w+\-].?)+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z\d\-]+\z/i
```